src/Supportive/Time/Stopwatch.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Supportive\Time;
+
+use function array_key_exists;
+
+final class Stopwatch
+{
+    /** @var array<non-empty-string, StartedPeriod> */
+    private array $periods = [];
+
+    public function __construct()
+    {
+    }
```

Comment:
Did you mean to leave this empty constructor here?

Reply:
Sorry, I intended to add dependencies, but I did not need them in the end, and I forgot to remove it. I just done it.